apollo/uma/queries.ts

```diff
@@ -21,12 +21,8 @@ export const EMP_DATA = gql`
           id
         }
         liquidationId
-        liquidator {
-          address
-        }
-        disputer {
-          address
-        }
+        liquidator
```

Comment:
I wouldn't merge this in yet until the mainnet subgraph is also fixed. It's currently syncing [here](https://thegraph.com/explorer/subgraph/nicholaspai/mainnet-contracts-staging?version=pending)